zipline/assets/asset_db_schema.py

```diff
@@ -42,16 +41,16 @@
     metadata,
     sa.Column(
         'sid',
-        sa.Integer,
+        sa.BigInteger,
         unique=True,
```

Comment:
Dates are saved as Integer, sqlite can handle both, postgres needs BigInt to process the dates. At this point we don't know which engine we're on. Biggest impact would be that asset-db get slightly bigger